docs/dev_quickstart.md

```diff
@@ -0,0 +1,117 @@
+# Quickstart / Pull request review
+
+Below the steps to checkout the code, optionally a specific branch, build, and then look at the current functionality.
+
+## Clone the code
+
+Clone latest version of sourcecode from github using [git](https://git-scm.com/downloads)
+
+```
+git clone git@github.com:molgenis/molgenis-emx2.git
+```
+
+Optionally, checkout the branch you like to review
+
+```
+cd molgenis-emx
+git checkout <branch name here>
+```
+
+Then you can either build + run the whole molgenis.jar, or only one app, described below. Or you can run inside
+IntelliJ.
+
+## Build whole system
+
+Requires [Postgresql 13](https://www.postgresql.org/download/) and java (we use
+adopt [OpenJDK 16](https://adoptopenjdk.net/)):
+
+* Optionally, drop a previous version of molgenis database (caution: destroys previous data!)
+  ```console
+  sudo -u postgres psql
```

Comment:
This is only relevant for Mac or Linux users. Or can you do this in powershell as well? I would imagine in Windows you would simply use the postgres app. I would suggest using docker serving the postgres backend. You can simply state `docker-compose up postgres` then you have a way of working agnostic to all platforms.

Reply:
I think this manual is meant for developers and those using Linux. If you are not able to do this, you shouldn't be a developer of the backend in my mind. We have docker compose for other users, incl frontend developers, If you are a windows user (I was one) I think you should figure this out yourself. 

So my proposed action is to make specific that this is for Linux/Mac users and that Windows users should tell us what needs to be different (I could ask Bart).